### DISCUSSION TITLE
Installation comprehensive test failure

### DISCUSSION THREAD

**Discussion by mdifab** [neuropoly/axondeepseg#735] [Help needed] [answered]
Hi,
I installed ADS and I tested the installation.
The quick test 'axondeepseg_test' was successful but the comprehensive test actually failed, returning:

<img width="1734" alt="Screenshot 2023-04-20 at 18 12 01" src="https://user-images.githubusercontent.com/119606403/233425368-bd0290d0-64fa-460d-9a07-1aaefc1c15f1.png">

I then installed Napari in the environment (ads_venv) within the folder axondeepseg but when I launched Napari and tried to select the ADS plugin from 'Plugins -> ADS plugin (napari-ads)' I actually don't have it in my list. Is it related to the fact that the comprehensive test failed? 

<img width="1044" alt="Screenshot 2023-04-20 at 18 14 50" src="https://user-images.githubusercontent.com/119606403/233426098-70778881-f12d-43f3-8440-adee5102e662.png">

Could you help me with both issues please? 

Thanks

**Comment by mathieuboudreau** [comment, marked as answer]
Hi @mdifab !

As for your first issue, it appears that the test data was not downloaded. This is usually done in the second installation step, with the command `pip install -e . plugins/`. Coincidentally, this is also the command that installs the plugin, so it's likely that this is the source of both your issues. Is it possible you missed this step, or that you didn't do it with your environment activated (`conda activate ads_venv`). Note that you also need to be in the axondeepseg/ repo when running this command.

**Reply by mdifab** [reply]
This solved the issue, thank you very much!
